### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).

It also includes the following components:

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)

* [Jamulus/Include-Server-Commands](https://hosted.weblate.org/projects/jamulus/include-server-commands/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: _translator-files/po/zh_CN/Command-Line-Options.po [hunk 1]
```diff
@@ -1,75 +1,75 @@
...
 msgid "en"
-msgstr ""
+msgstr "zh_CN"
 
 #. type: YAML Front Matter: layout
 #: ../wiki/en/Command-Line-Options.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "维基"
```
Comment (ann0see) [inline _translator-files/po/zh_CN/Command-Line-Options.po]:
```suggestion
msgstr "wiki"
```

**Review by ann0see — CHANGES_REQUESTED** [review]

File: _translator-files/po/zh_CN/Command-Line-Options.po (hunk 1 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/Command-Line-Options.po]: Corrected on weblate

**Comment by yangyangdaji** [comment]
i'm ready

**Comment by ignotus666** [comment]
> i'm ready

@yangyangdaji Wow, that was really fast, thanks! I'm testing the website locally and for some reason zh_CN isn't working so I'm looking into that. Meanwhile, could you please review the links that are for specific paragraphs (I'll point one out to you in this PR). You can find info on how they have to be translated [here](https://github.com/jamulussoftware/jamuluswebsite/tree/release/_translator-files).

**Comment by yangyangdaji** [comment]
OK

File: _translator-files/po/zh_CN/FAQ.po [hunk 2]
```diff
@@ -46,159 +47,161 @@ msgid ""
...
 #: ../wiki/en/FAQ.md:26
 msgid "No. Musicians on a Jamulus Server play in real time together as they would in person. If you want to have a time signal, then there are shared metronome solutions on the Internet you can try. But it’s probably best to just minimise latency so nobody has an overall delay more than about 30-50 ms."
-msgstr ""
+msgstr "不会。Jamulus 服务器上的音乐家可以实时一起演奏，就像他们亲自演奏一样。如果你想有时间信号，那么网上有共享节拍器的解决方案你可以试试。但最好只是尽量减少延迟，这样就没有人有超过 30-50 毫秒的整体延迟。"
 
 #. type: Plain text
 #: ../wiki/en/FAQ.md:28
 msgid "Bear in mind also that all participants should follow [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) which will also determine whether you can play in time properly."
-msgstr ""
+msgstr "还请记住，所有参与者都应遵循 [黄金法则](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together) 这也将决定您是否可以使用及时正确。"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/FAQ.po]: This is an example of one of those links. The part after the hashtag should be translated, using the exact same wording as the heading in `Client-Troubleshooting`. On the other hand, I don't have a clue about how punctuation works in Chinese - there don't seem to be spaces - so I'm not sure if everything should be joined up by hyphens or they're only inserted where there are commas or other punctuation marks. Once I get the website working I'll look into it.

File: _translator-files/po/zh_CN/1-index.po [hunk 3]
```diff
@@ -35,24 +35,39 @@ msgid ""
 "mTOtherPlatforms: 'other platforms'\n"
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"lang: \"zh_CN\"\n"
+"永久性的。/\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/1-index.po]: This should be left as `permalink:`

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/1-index.po]: Fixed. Wait for sync

File: _translator-files/po/zh_CN/Installation-for-Windows.po [hunk 4]
```diff
@@ -1,289 +1,290 @@
...
 #: ../wiki/en/Installation-for-Windows.md:26
 msgid "_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._"
-msgstr ""
+msgstr "_请注意，在未获得 [Steinberg 的许可协议](https://www.steinberg.net/developers/) 的情况下，您不得重新分发此二进制文件。_"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Windows.md:28
 msgid "If you don't want to use ASIO or prefer [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html), you can also use the [JACK version of Jamulus]({{ site.download_root_link }}{{ site.download_file_names.windows-jack }})."
-msgstr ""
+msgstr "如果您不想使用 ASIO 或者更喜欢 [JACK on Windows](https://jackaudio.org/faq/jack_on_windows.html)，也可以使用 [JACK 版本的 Jamulus]({{ site.download_root_link } {{ site.download_file_names.windows-jack }}）。"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Installation-for-Windows.po]: There's a closing curly bracket missing in `{{ site.download_root_link }`.

File: _translator-files/po/zh_CN/Running-a-Server.po [hunk 5]
```diff
@@ -173,16 +176,20 @@ msgid ""
 "<figcaption>How Registered Servers work</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Diagram of connections between Clients within a Jamulus Registered Server\">\n"
+"<figcaption>How Registered Servers work</figcaption>\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Running-a-Server.po]: Preferably, the figcaption text should be translated.

File: _translator-files/po/zh_CN/Installation-for-Windows.po (hunk 4 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Installation-for-Windows.po]: Fixed For now

File: _translator-files/po/zh_CN/Running-a-Server.po [hunk 6]
```diff
@@ -193,111 +200,115 @@ msgid ""
...
 #: ../wiki/en/Running-a-Server.md:86
 msgid "### Installation"
-msgstr ""
+msgstr "### 安装"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:88
 msgid "Download the [latest headless (amd64) .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) or, if you use a Raspberry Pi etc. download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or the [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }})"
-msgstr ""
+msgstr "下载 [最新的无头 (amd64) .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }}) 或者，如果您使用 Raspberry Pi 等，请下载 [最新的 armhf .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) 或 [最新的 arm64 .deb 文件]({{ site.download_root_link }}{{ site.download_file_names.deb-headless- arm64 }})"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Running-a-Server.po]: There's an unwanted space in `site.download_file_names.deb-headless- arm64` just before `arm64`.

File: _translator-files/po/zh_CN/1-index.po [hunk 7]
```diff
@@ -35,24 +35,39 @@ msgid ""
 "mTOtherPlatforms: 'other platforms'\n"
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
+"lang: \"zh_CN\"\n"
+"permalink: /\n"
+"layout: mainhomepage\n"
+"ldjAppCategory: \"音乐\"\n"
+"ldjSlogan:\"在线播放音乐。与朋友一起。免费的。\"\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/1-index.po]: There should be a space after the colon.

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/1-index.po]: OK

File: _translator-files/po/zh_CN/1-index.po [hunk 8]
```diff
@@ -35,24 +35,39 @@ msgid ""
 "mTOtherPlatforms: 'other platforms'\n"
 "---\n"
 msgstr ""
+"---\n"
+"title: \"Jamulus - 在线播放音乐。与朋友一起。免费的。\"\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/1-index.po:39]: Jekyll keeps complaining about this line but I can't see what's wrong with it. The error message is:
`did not find expected key while parsing a block mapping at line 2 column 1`
@ann0see any ideas?

Comment (ignotus666) [reply _translator-files/po/zh_CN/1-index.po:39]: Ok, got it. Missing quotation marks further below.

File: _translator-files/po/zh_CN/1-index.po [hunk 9]
```diff
@@ -35,24 +35,39 @@ msgid ""
...
+"permalink: /\n"
+"layout: mainhomepage\n"
+"ldjAppCategory: \"音乐\"\n"
+"ldjSlogan:\"在线播放音乐。与朋友一起。免费的。\"\n"
+"ldjCreator: \"Volker Fischer 和贡献者\"\n"
+"metadescription: \"Jamulus 是一个开源软件，让音乐家在互联网上实时演奏音乐，排练或只是一起演奏。\"\n"
+"mAltProgIcon: \"Jamulus 图标\"\n"
+"mTSlogan: \"在线播放音乐。与朋友一起。免费的。\"\n"
+"mTGetStartedNow: \"现在就开始吧！\"\n"
+"mTDownloadNow: \"现在下载\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/1-index.po]: Missing quotation mark at the end of `现在下载`. The original in EN uses single quotes here - not sure if it's important but no harm in keeping it that way.

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/1-index.po]: ok

File: _translator-files/po/zh_CN/1-index.po (hunk 7 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/1-index.po]: Wait for sync

File: _translator-files/po/zh_CN/1-index.po [hunk 10]
```diff
@@ -35,24 +35,39 @@ msgid ""
...
+"mTOtherPlatforms: \"其他平台\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/1-index.po]: Closing quote missing here too.

This is the last error I get and when corrected the website works fine in Chinese : ) All that's left are the links, which I'll test now to see how they should be done. Oh, and screenshots, which would be cool to have too. Just ask if in doubt about how to contribute them.

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/1-index.po]: OK

File: _translator-files/po/zh_CN/FAQ.po (hunk 2 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/FAQ.po]: Hmm, I'm struggling with this; please don't change these links yet until I figure out how they have to be done in Chinese.

File: _translator-files/po/zh_CN/Software-Manual.po [hunk 11]
```diff
@@ -91,364 +94,368 @@ msgid ""
 "\t<figcaption>Your local mix when connected to a Server</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"<img src=\"{% include img/en-screenshots/main-screen-medium.inc %}\" style=\"border: 5px solid gray;\" loading=\"lazy\" alt=\"Jamulus 主窗口的图像\">\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Software-Manual.po]: Sorry, I overlooked this - you need to change the language code in `en-screenshot` to `zh_CN-screenshot`. That's for all screenshots.

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Software-Manual.po]: I also know, but may not find the file.

File: _translator-files/po/zh_CN/FAQ.po (hunk 2 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/FAQ.po]: Ok, so the above link works when written as follows:

`[黄金法则](/wiki/Client-Troubleshooting#你们听起来都不错但很难保持在一起)`. No hyphens.

File: _translator-files/po/zh_CN/Software-Manual.po (hunk 11 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/Software-Manual.po]: Yes, it will. The zh_CN folder for screenshots contains the ones in EN until they are replaced.

Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Software-Manual.po]: done

File: _translator-files/po/zh_CN/Running-a-Server.po (hunk 5 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Running-a-Server.po]: done

File: _translator-files/po/zh_CN/Include-Server-Commands.po [hunk 12]
```diff
@@ -1,111 +1,111 @@
...
 #: ../wiki/en/Include-Server-Commands.md:16
 msgid "`-T` or `--multithreading` Use multithreading to make better use of multi-core CPUs to support more Clients"
-msgstr ""
+msgstr "`-T` 或 `--multithreading` 使用多线程更好地利用多核 CPU 来支持更多的客户端"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:17
 msgid "`-u` or `--numchannels` Maximum number of channels (Clients)"
-msgstr ""
+msgstr "“-u”或“--数字通道” 最大通道数（客户端）"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Include-Server-Commands.po]: Wrong quotes used.

File: _translator-files/po/zh_CN/Include-Shared-Commands.po [hunk 13]
```diff
@@ -1,81 +1,81 @@
...
 msgid "[comment]: # (This is an include file for use in multiple documents)"
-msgstr ""
+msgstr "[comment]: # (这是一个用于多个文档的包含文件)"
 
 #. type: Plain text
 #: ../wiki/en/Include-Shared-Commands.md:5
 #, no-wrap
 msgid "`-h` or `--help`           Display help text         \n"
-msgstr ""
+msgstr "“-h”或“--help” 显示帮助文本         \n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Include-Shared-Commands.po]: Quotes

File: _translator-files/po/zh_CN/Include-Server-Commands.po (hunk 12 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Include-Server-Commands.po]: Fixed

File: _translator-files/po/zh_CN/Include-Shared-Commands.po (hunk 13 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Include-Shared-Commands.po]: Fixed

File: _translator-files/po/zh_CN/Running-a-Server.po (hunk 6 above)
Comment (yangyangdaji) [reply _translator-files/po/zh_CN/Running-a-Server.po]: ok

File: _translator-files/po/zh_CN/Software-Manual.po [hunk 14]
```diff
@@ -91,364 +94,368 @@ msgid ""
...
 #: ../wiki/en/Software-Manual.md:92
 msgid "## Server audio mixer"
-msgstr ""
+msgstr "## 服务器混音器"
 
 #. type: Plain text
 #: ../wiki/en/Software-Manual.md:94
 msgid "<figure><img src=\"{% include img/en-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"Image of a pair of server mixer controls\"></figure>"
-msgstr ""
+msgstr "<figure><img src=\"{% include img/zh_CN-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"一对服务器混音器控件的图像/figure>"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Software-Manual.po]: `\"><` missing at end of line before `/figure`.

File: _translator-files/po/zh_CN/Software-Manual.po [hunk 15]
```diff
@@ -91,364 +94,368 @@ msgid ""
...
 #: ../wiki/en/Software-Manual.md:92
 msgid "## Server audio mixer"
-msgstr ""
+msgstr "## 服务器混音器"
 
 #. type: Plain text
 #: ../wiki/en/Software-Manual.md:94
 msgid "<figure><img src=\"{% include img/en-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"Image of a pair of server mixer controls\"></figure>"
-msgstr ""
+msgstr "<figure><img src=\"{% include img/zh_CN-screenshots/mixer-channels.inc %}\" style=\"float:left; margin-right:10px; margin-bottom:20px; border: 5px solid grey;\" loading=\"lazy\" alt=\"一对服务器混音器控件的图像></figure>"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Software-Manual.po]: There's still a closing quotation mark missing.

File: _translator-files/po/zh_CN/Client-Troubleshooting.po [hunk 16]
```diff
@@ -46,236 +47,238 @@ msgid ""
...
 msgid "Start with the simple stuff: make sure your instrument/microphone and headphones are connected to the correct sockets. Make sure no other applications like your browser, video conferencing tool etc. is also using your sound card. You should shut those down when using Jamulus. If that all looks OK and the problem persists, it's likely there's an issue with your sound device settings. This will depend on your particular setup (platform, hardware, software and drivers), so it's best to ask on [the forums](https://github.com/jamulussoftware/jamulus/discussions) for guidance."
-msgstr ""
+msgstr "从简单的东西开始:确保您的乐器/麦克风和耳机连接到正确的插座。确保没有其他应用程序（例如您的浏览器、视频会议工具等）也在使用您的声卡。使用 Jamulus 时应该关闭它们。如果一切正常且问题仍然存在，则可能是您的声音设备设置存在问题。这取决于您的特定设置（平台、硬件、软件和驱动程序），因此最好在 [论坛](https://github.com/jamulussoftware/jamulus/discussions) 上寻求指导。"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:25
 #, no-wrap
 msgid "**Windows users (ASIO4ALL)**: If you’re using the ASIO4ALL driver have a look at the [ASIO4ALL setup section](Installation-for-Windows#setting-up-asio4all)\n"
-msgstr ""
+msgstr "**Windows 用户 (ASIO4ALL)**:如果您使用的是 ASIO4ALL 驱动程序，请查看 [ASIO4ALL 设置部分](Installation-for-Windows#设置 ASIO4ALL)\n"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Client-Troubleshooting.po]: In these links there can't be any capital letters nor spaces. It seems that if all the characters are Chinese, as there are no spaces, hyphens aren't required either. But in this case, as there's a space, it should be replaced with a hyphen - so it should look like this:

`(Installation-for-Windows#设置-asio4all)`.

These links are where most people stumble, and add Chinese characters into the mix and it gets even more complicated ;)

File: _translator-files/po/zh_CN/Client-Troubleshooting.po [hunk 17]
```diff
@@ -46,236 +47,238 @@ msgid ""
...
 #: ../wiki/en/Client-Troubleshooting.md:78
 msgid "In the UK (and possibly other regions/routers) the Virgin Media Cable Internet Modem setting can cause an issue. \"Block Fragmented IP Packets\" should not be checked. For other routers/ISP, also try turning off SPI (Stateful Packet Inspection) and if that fixes it, you can make a judgment about whether to leave that off or not."
-msgstr ""
+msgstr "在英国（可能还有其他区域/路由器），维珍媒体有线互联网调制解调器设置可能会导致问题。不应检查“阻止分段 IP 数据包”。对于其他路由器/ISP，也可以尝试关闭 SPI（状态数据包检测），如果修复了它，您可以判断是否将其关闭。"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:80
 msgid "In some cases, it may be your ISP that is blocking your use of Jamulus. See the note on the [server troubleshooting page](Server-Troubleshooting#nobody-can-connect-to-my-server---but-i-can-connect-locally) about \"Nobody can connect to my Server\""
-msgstr ""
+msgstr "在某些情况下，可能是您的 ISP 阻止了您使用 Jamulus。请参阅 [服务器故障排除页面](Server-Troubleshooting#没有人可以连接到我的服务器 - 但我可以在本地连接)上关于“没有人可以连接到我的服务器”的注释"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Client-Troubleshooting.po]: No spaces allowed in link. It should look like this:

`(Server-Troubleshooting#没有人可以连接到我的服务器---但我可以在本地连接)`

File: _translator-files/po/zh_CN/Server-Troubleshooting.po [hunk 18]
```diff
@@ -51,113 +52,115 @@ msgid ""
...
 #: ../wiki/en/Server-Troubleshooting.md:39
 msgid "### What address do I give to people so that they can connect to my Server?"
-msgstr ""
+msgstr "### 我应该给人们什么地址，以便他们可以连接到我的服务器？"
 
 #. type: Plain text
 #: ../wiki/en/Server-Troubleshooting.md:41
 msgid "This should be your **public** IP address (find that with [Google](https://www.google.com/search?q=whatsmyip)). Connect your **own** Jamulus Client to the **local** IP address of your Server (`localhost` or `127.0.0.1` if it's on the same machine as your Client). Note that your public IP address can change - see the note on \"dynamic DNS\" in [this guide](Running-a-Server#dynamic-dns-and-why-you-will-probably-need-it)."
-msgstr ""
+msgstr "这应该是您的**公共** IP 地址（通过 [Google](https://www.google.com/search?q=whatsmyip) 查找）。将您的**自己的** Jamulus 客户端连接到服务器的**本地** IP 地址（`localhost` 或`127.0.0.1`，如果它与您的客户端在同一台机器上）。请注意，您的公共 IP 地址可能会更改 - 请参阅 [本指南] 中有关“动态 DNS”的说明（运行服务器#动态 DNS 以及您可能需要它的原因）。"
```
Comment (ignotus666) [inline _translator-files/po/zh_CN/Server-Troubleshooting.po]: The structure for links is: `[some text](A-Document#paragraph-header)`. The content in square brackets should not be separated from the link in normal brackets. I don't know how you would have to reformulate the sentence to maintain this rule. Also, spaces in`（运行服务器#动态 DNS 以及您可能需要它的原因）` should be replaced with hyphens.

Comment (ignotus666) [reply _translator-files/po/zh_CN/Server-Troubleshooting.po]: What I mean is that it should look like this:

`请注意，您的公共 IP 地址可能会更改 - 请参阅 [本指南](Running-a-Server#运行服务器#动态-DNS-以及您可能需要它的原因) 中有关“动态 DNS”的说明。` 

It's missing `Running-a-Server#` (isn't it?)

**Comment by ann0see** [comment]
@ignotus666 @yangyangdaji This still seems to be blocking. When can we plan the release?

**Comment by ignotus666** [comment]
I think it's just these last couple of corrections that are missing to make it 100% (or near enough), though I'm not sure if there are many more of those links to update. The website works fine as it is though.

**Comment by ann0see** [comment]
Ok. Thanks for testing. It probably makes sense to wait on the link fixes then. But feel free to merge this PR.

**Comment by yangyangdaji** [comment]
@ignotus666 @ann0see I found some problems already fixed for now on Weblate 
Wait for sync and review

File: _translator-files/po/zh_CN/Client-Troubleshooting.po (hunk 17 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/Client-Troubleshooting.po]: Corrected this myself.

File: _translator-files/po/zh_CN/Server-Troubleshooting.po (hunk 18 above)
Comment (ignotus666) [reply _translator-files/po/zh_CN/Server-Troubleshooting.po]: Corrected this too.

**Comment by ignotus666** [comment]
Ok, so I did another quick review and corrected a few things I found myself to speed things up. I'll merge it now so the website can go live when you see fit @ann0see.

@yangyangdaji when it goes live (should be soon) it would be great if you could go through the website to check if everything works (links) and is displayed correctly (formatting). I think it all looks ok but since I can't read Chinese I can't really be sure.

**Comment by yangyangdaji** [comment]
ok @ignotus666 Thanks for help me

**Comment by yangyangdaji** [comment]
@ignotus666 how do test

**Comment by ignotus666** [comment]
You'll have to either wait until the website goes live, or you can test locally - but you will need to install Jekyll. You can find instructions [here](https://github.com/jamulussoftware/jamuluswebsite#viewing-the-website-offline).

**Comment by yangyangdaji** [comment]
ok

**Pull request merged**